wave-api/src/test/groovy/io/seqera/wave/api/FusionVersionTest.groovy

```diff
@@ -30,18 +30,23 @@ class FusionVersionTest extends Specification {
         FusionVersion.from(STR) == EXPECTED
 
         where:
-        EXPECTED                            | STR
-        null                                | null
-        new FusionVersion('2','amd64')      | 'https://foo.com/v2-amd64.json'
-        new FusionVersion('22','amd64')     | 'https://foo.com/v22-amd64.json'
-        new FusionVersion('2.1','amd64')    | 'https://foo.com/v2.1-amd64.json'
-        new FusionVersion('2.11','amd64')   | 'https://foo.com/v2.11-amd64.json'
-        new FusionVersion('2.1.3','amd64')  | 'https://foo.com/v2.1.3-amd64.json'
-        new FusionVersion('2.1.3','arm64')  | 'https://foo.com/v2.1.3-arm64.json'
-        new FusionVersion('2.1.33','arm64') | 'https://foo.com/v2.1.33-arm64.json'
-        new FusionVersion('2.1.3a','arm64') | 'https://foo.com/v2.1.3a-arm64.json'
-        new FusionVersion('2.1.3.a','arm64')| 'https://foo.com/v2.1.3.a-arm64.json'
-
+        EXPECTED                                | STR
+        null                                    | null
+        new FusionVersion('2','amd64')          | 'https://foo.com/v2-amd64.json'
+        new FusionVersion('22','amd64')         | 'https://foo.com/v22-amd64.json'
+        new FusionVersion('2.1','amd64')        | 'https://foo.com/v2.1-amd64.json'
+        new FusionVersion('2.11','amd64')       | 'https://foo.com/v2.11-amd64.json'
+        new FusionVersion('2.1.3','amd64')      | 'https://foo.com/v2.1.3-amd64.json'
+        new FusionVersion('2.1.3','arm64')      | 'https://foo.com/v2.1.3-arm64.json'
+        new FusionVersion('2.1.33','arm64')     | 'https://foo.com/v2.1.33-arm64.json'
+        new FusionVersion('2.1.3a','arm64')     | 'https://foo.com/v2.1.3a-arm64.json'
+        new FusionVersion('2.1.3.a','arm64')    | 'https://foo.com/v2.1.3.a-arm64.json'
+        new FusionVersion('2.2.8','amd64')      | 'https://foo.com//releases/pkg/2/2/8/fusion-amd64.tar.gz'
+        new FusionVersion('2.2.8','arm64')      | 'https://foo.com//releases/pkg/2/2/8/fusion-arm64.tar.gz'
+        new FusionVersion('2.2','arm64')        | 'https://foo.com//releases/pkg/2/2/fusion-arm64.tar.gz'
+        new FusionVersion('2.22','arm64')       | 'https://foo.com//releases/pkg/2/22/fusion-arm64.tar.gz'
+        new FusionVersion('2.2.33','arm64')     | 'https://foo.com//releases/pkg/2/2/33/fusion-arm64.tar.gz'
+        new FusionVersion('2.2.8.a','arm64')    | 'https://foo.com//releases/pkg/2/2/8/a/fusion-arm64.tar.gz'
```

Comment:
Something has changed in the URL schema? tagging @jordeu for visibility

Reply:
in here we use location to parse the fusion version
https://github.com/seqeralabs/libseqera/blob/88d7fac231b0c0c18a9a39553eff0e2ccc45f5b5/wave-api/src/main/java/io/seqera/wave/api/ContainerConfig.java#L99

and if you check inside the json 
https://fusionfs.seqera.io/releases/v2.2.8-amd64.json
location is "https://fusionfs.seqera.io/releases/pkg/2/2/8/fusion-amd64.tar.gz"

Reply:
I see. `https://foo.com//releases ` should be `https://foo.comreleases`

Reply:
oh sorry, fixed

Reply:
We didn't change anything related to this. What's the problem?

Reply:
@jordeu I don't think there is a change from your side
its because we are parsing a JSON file but passing fusion location to this parser, which is a tar.gz file